src/frontend/css/style.scss

```diff
@@ -142,14 +142,12 @@ label:not([for]) {
   @apply mb-2 text-gray-600;
 }
 
-.table-header {
-  .table-search {
-    svg {
-      width: 16px;
-      height: 16px;
-      top: 14px;
-      left: 14px;
-    }
+#table-search {
+  svg {
+    width: 16px;
+    height: 16px;
+    top: 14px;
+    left: 14px;
   }
 }
```

Comment:
```suggestion
```
This can be deleted as mentioned earlier. Additionally, the lines
```
  &.rounded-r-none {
    border-top-right-radius: 0;
    border-bottom-right-radius: 0;
  }
```
should be added in line 95 (the definitions for form elements).